### PR TITLE
RC: Limiting the number of databases per plan to only one database

### PR DESCRIPTION
**Pull request by cmilesb**
Ticket: DOC-3134
Staging link: 
https://docs.redis.com/staging/DOC-3134/rc/subscriptions/create-fixed-subscription/
https://docs.redis.com/staging/DOC-3134/rc/changelog/january-2024/
https://docs.redis.com/staging/DOC-3134/rc/subscriptions/view-fixed-subscription/#upgrade-subscription-plan

### CODE REVIEW

**Comment by cmilesb** [comment]
Approved via slack by Adi Abiri Silberman.

File: content/rc/subscriptions/create-fixed-subscription.md [hunk 1]
```diff
@@ -19,16 +19,16 @@ If you're new to Redis Cloud, the [quick start]({{<relref "/rc/rc-quickstart.md"
 
 ## Fixed plan subscription tiers
 
-Fixed plan pricing scales according to the memory size of all databases defined in the subscription.  Additional limits also apply, as shown here (updated November 2022):
+Fixed plan pricing scales according to the memory size of the database defined in the subscription.  Additional limits also apply, as shown here (updated January 2023):
```
Comment (dwdougherty) [inline content/rc/subscriptions/create-fixed-subscription.md:22]: Should this be "January 2024"?

**Review by dwdougherty — CHANGES_REQUESTED** [review]
Apart from my one question/comment, LGTM.

File: content/rc/subscriptions/create-fixed-subscription.md (hunk 1 above)
Comment (cmilesb) [reply content/rc/subscriptions/create-fixed-subscription.md:22]: Yup! Thanks for catching that.
```suggestion
Fixed plan pricing scales according to the memory size of the database defined in the subscription.  Additional limits also apply, as shown here (updated January 2024):
```

**Review by dwdougherty — APPROVED** [review]
LGTM

**Pull request merged**